types/adm-zip/index.d.ts

```diff
@@ -282,6 +363,160 @@ declare namespace AdmZip {
         fnameLen: number;
         extraLen: number;
     }
+
+    interface Constants {
+        /* The local file header */
+        LOCHDR: 30; // LOC header size
+        LOCSIG: 0x04034b50; // "PK\003\004"
+        LOCVER: 4;	// version needed to extract
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/adm-zip/index.d.ts;
  Line: 371;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.